x/group/keeper/keeper.go

```diff
@@ -73,16 +73,26 @@ type Keeper struct {
 	voteByVoterIndex    orm.Index
 
 	router *authmiddleware.MsgServiceRouter
+
+	maxMetadataLength uint64
```

Comment:
In gov, I named this `maxMetadataLen`.

It would be a good idea to use the same. (i prefer shorter)